telegram/_bot.py

```diff
@@ -5612,6 +5613,9 @@ def __eq__(self, other: object) -> bool:
     def __hash__(self) -> int:
         return hash(self.bot)
 
+    def __reduce__(self):
+        raise pickle.PicklingError('Bot objects cannot be pickled!')
+
```

Comment:
I'm not sure if this necessary. The default-implementation is already not picklable because locks are used in the request object (both with urllib3 and with httpx). This is also highly likely to be the case for any custom implementation of `BasePersistence` on v14.
Apart from that, users can still override this to eg skip the request object on pickle.dumps …

Reply:
ah, in the mock-up I didn't want to import `telegram.Bot`, so I manually produced an exception.
Anyway, I'd also be okay with keeping it, just wanted to point out the culprits